src/core/constants.js

```diff
@@ -11,6 +11,7 @@ var PI = Math.PI;
 module.exports = {
   // GRAPHICS RENDERER
   /**
+   * Denotes the default two-dimensional renderer.
```

Comment:
Wording wise "denote" may be a bit difficult for people whose English is not their first language.